### PR TITLE
Fix image prune --filter cmd behavior

### PR DESCRIPTION
**Pull request by jmguzik**
Glue code added to push filters to the prune image engine method.

### CODE REVIEW

File: test/e2e/images_test.go [hunk 1]
```diff
@@ -425,4 +425,20 @@ LABEL "com.example.vendor"="Example Vendor"
...
+		dockerfile2 := `FROM quay.io/libpod/alpine:latest
+RUN > file2
+`
+		podmanTest.BuildImageWithLabel(dockerfile, "foobar.com/workdir:latest", "false", "abc")
+		podmanTest.BuildImageWithLabel(dockerfile2, "foobar.com/workdir:latest", "false", "xyz")
+		// --force used to to avoid y/n question
+		result := podmanTest.Podman([]string{"image", "prune", "--filter", "label=abc", "--force"})
+		result.WaitWithDefaultTimeout()
+		Expect(result).Should(Exit(0))
+		Expect(len(result.OutputToStringArray())).To(Equal(1))
```
Comment (rhatdan) [inline test/e2e/images_test.go:441]: Should check to make sure the image is NOT apc.

Comment (jmguzik) [reply test/e2e/images_test.go:441]: what does apc mean?

Comment (rhatdan) [reply test/e2e/images_test.go:441]: typo, should have been "abc"

**Comment by rhatdan** [comment]
@jmguzik Still working ont his?

**Comment by jmguzik** [comment]
> @jmguzik Still working ont his?

Yes, but not enough time this week.

**Review by vrothberg — COMMENTED** [review]
Drop-by-comment: please update the commit message to describe which issue it's fixing.

**Comment by jmguzik** [comment]
> Drop-by-comment: please update the commit message to describe which issue it's fixing.

There is no particular issue. I just found the error, not reported it as an issue, instead opened a PR.

File: test/e2e/images_test.go (hunk 1 above)
Comment (jmguzik) [reply test/e2e/images_test.go:441]: Added check 3 lines below

**Comment by vrothberg** [comment]
> > Drop-by-comment: please update the commit message to describe which issue it's fixing.
> 
> There is no particular issue. I just found the error, not reported it as an issue, instead opened a PR.

The challenge I have as a reader/reviewer of the commit message is that it mentions to fix something but it doesn't state what the problem actually was.

**Comment by jmguzik** [comment]
> The challenge I have as a reader/reviewer of the commit message is that it mentions to fix something but it doesn't state what the problem actually was.

Added commit description with further info about the problem.

**Review by vrothberg — COMMENTED** [review]
LGTM, thanks

**Comment by rhatdan** [comment]
/approve
/lgtm
/hold
Thanks @jmguzik

**Pull request merged**